h2o-py/h2o/explanation/_explain.py

```diff
@@ -1198,6 +1297,8 @@ def pd_plot(
         save_plot_path=None, # type: Optional[str]
         binary_response_scale="response", # type: Literal["response", "logodds"]
         grouping_column=None,  # type: Optional[str]
+        output_graphing_data=False,  # type: bool
+        nbins = 100,  # type: int
         **kwargs
```

Comment:
(Not for this PR) Since we are using `**kwargs` I think we should check if the user specified some undefined argument and if so warn the user. (E.g. using `row` instead of `row_index` would create a PDP instead of ICE and could be hard to find what's wrong).

Reply:
[a9dfa0d](https://github.com/h2oai/h2o-3/pull/6119/commits/a9dfa0dca78c37d998caa4dbd8d975cb361b9ea8)